Model/Currency.php

```diff
@@ -212,16 +213,37 @@ protected function getConsumerIp()
     }
 
     /**
-     * Check IP is local ip
-     *
-     * When running in Docker, the IP addresses assigned by the Docker service
-     * must be added here in order for the application to run. Error displayed
-     * is regarding Line 71 in Model/Reach.php.
-     *
+     * check IP is local machine IP
+     * @param  string $ip
      * @return boolean
      */
     protected function checkLocalIP($ip)
     {
-        return in_array($ip, ['localhost','127.0.0.1','172.19.0.1','172.25.0.7','172.25.0.1']);
+        //Ryan's code from MAG-75 (to be able to test payment code properly)
```

Comment:
this comment was definitely helpful for me, thank you! Could you remove it before we push the code? I don't think this would benefit anyone other than me as a reviewer when looking at this code.

Reply:
I can definitely remove the comment - only thing is I wanted to make sure that I do not get credit for someone else's work. Thank you.

Reply:
Commit 55d3c65 
Thanks.